net/keepalived/files/keepalived.init

```diff
@@ -293,7 +293,7 @@ vrrp_instance() {
 		no_val_use_vmac
 
 	print_notify "INSTANCE" "$name" notify_backup notify_master \
-		notify_fault notify_stop notify
```

Comment:
Is notify no longer officially supported? Can you show an example of the warnings you've mentioned?

I understand there's new requirements with regards to config options `script_user` and `enable_script_security` which will likely generate warnings as we're not handling them in this init script.

Reply:
without notify:

> Wed Jan 10 13:14:12 2018 local1.info Keepalived_vrrp[3202]: Opening file '/tmp/keepalived.conf'.
> Wed Jan 10 13:14:12 2018 local1.info Keepalived_vrrp[3202]: WARNING - default user 'keepalived_script' for script execution does not exist - please create.
> Wed Jan 10 13:14:12 2018 local1.info Keepalived_vrrp[3202]: SECURITY VIOLATION - scripts are being executed but script_security not enabled.
> Wed Jan 10 13:14:12 2018 local1.info Keepalived_vrrp[3202]: Using LinkWatch kernel netlink reflector...
> Wed Jan 10 13:14:12 2018 local1.info Keepalived_vrrp[3202]: VRRP_Instance(vrrp) Entering BACKUP STATE
> Wed Jan 10 13:14:12 2018 local1.debug Keepalived_vrrp[3202]: Opening script file /bin/busybox
> Wed Jan 10 13:14:15 2018 local1.info Keepalived_vrrp[3202]: VRRP_Instance(vrrp) Transition to MASTER STATE
> Wed Jan 10 13:14:16 2018 local1.info Keepalived_vrrp[3202]: VRRP_Instance(vrrp) Entering MASTER STATE
> Wed Jan 10 13:14:16 2018 local1.debug Keepalived_vrrp[3202]: Opening script file /bin/busybox
> Wed Jan 10 13:15:21 2018 daemon.info Keepalived[3077]: Stopping
> Wed Jan 10 13:15:21 2018 local1.info Keepalived_vrrp[3202]: VRRP_Instance(vrrp) sent 0 priority
> Wed Jan 10 13:15:22 2018 local1.info Keepalived_vrrp[3202]: Stopped
> Wed Jan 10 13:15:22 2018 daemon.info Keepalived[3077]: Stopped Keepalived v1.3.9 (12/18,2017), git commit v17.01.4-185-g143cbb5

with notify:

> Wed Jan 10 13:15:26 2018 local1.info Keepalived_vrrp[16882]: Opening file '/tmp/keepalived.conf'.
> Wed Jan 10 13:15:26 2018 local1.info Keepalived_vrrp[16882]: WARNING - default user 'keepalived_script' for script execution does not exist - please create.
-----------
> Wed Jan 10 13:15:26 2018 local1.info Keepalived_vrrp[16882]: Script /bin/busybox env -i ACTION=NOTIFY TYPE=INSTANCE NAME=vrrp /sbin/hotplug-call keepalived cannot be accessed - No such file or directory
> Wed Jan 10 13:15:26 2018 local1.info Keepalived_vrrp[16882]: Disabling notify script /bin/busybox env -i ACTION=NOTIFY TYPE=INSTANCE NAME=vrrp /sbin/hotplug-call keepalived since not found/accessible
-----------
> Wed Jan 10 13:15:26 2018 local1.info Keepalived_vrrp[16882]: SECURITY VIOLATION - scripts are being executed but script_security not enabled.
> Wed Jan 10 13:15:26 2018 local1.info Keepalived_vrrp[16882]: Using LinkWatch kernel netlink reflector...
> Wed Jan 10 13:15:26 2018 local1.info Keepalived_vrrp[16882]: VRRP_Instance(vrrp) Entering BACKUP STATE
> Wed Jan 10 13:15:26 2018 local1.debug Keepalived_vrrp[16882]: Opening script file /bin/busybox
> Wed Jan 10 13:15:30 2018 local1.info Keepalived_vrrp[16882]: VRRP_Instance(vrrp) Transition to MASTER STATE
> Wed Jan 10 13:15:31 2018 local1.info Keepalived_vrrp[16882]: VRRP_Instance(vrrp) Entering MASTER STATE
> Wed Jan 10 13:15:31 2018 local1.debug Keepalived_vrrp[16882]: Opening script file /bin/busybox

All other scripts work as expected. I am not sure why keepalived does not recognize this so i thought it is not supported any more. But i have nothing found in the documentation of keepalived.
If i add a group notify is supported.
?